hadoop-ozone/dist/pom.xml

```
@@ -105,6 +105,16 @@
                   <artifactId>protobuf-java</artifactId>
                   <version>${grpc.protobuf-compile.version}</version>
                 </artifactItem>
+                <artifactItem>
+                  <groupId>org.apache.hadoop</groupId>
+                  <artifactId>hadoop-auth</artifactId>
+                  <version>2.7.3</version>
+                </artifactItem>
+                <artifactItem>
+                  <groupId>org.apache.hadoop</groupId>
+                  <artifactId>hadoop-annotations</artifactId>
+                  <version>2.7.3</version>
```

Same. Try to use `hadoop2.version` instead?

```suggestion
                  <version>${hadoop2.version}</version>
```